addon/ng2/blueprints/ng2/files/__path__/polyfills.ts

```diff
@@ -0,0 +1,4 @@
+// Prefer CoreJS over the polyfills above
+import 'core-js/es6';
+import 'core-js/es7/reflect';
+import 'zone.js/dist/zone';
```

Comment:
i'm still not comfortable with this being import/require'd yet, there's an increasing number of Zone issues we're seeing with people require'ing Zone.